### DISCUSSION TITLE
forcing light mode with autoSwitchAppearance

### DISCUSSION THREAD

**Discussion by benmatheson** [jpanther/congo#148] [General]
Hi, I love Congo and was very excited to upgrade to version 2. 

I'm trying to force "light mode" but am experiencing the page switching to dark mode on a Mac device that has dark mode in the OS settings. I'm wondering if it's possible to override the OS settings, or whether those will prevail. A [live version is here.](https://benmatheson.github.io/)

My params.toml looks like this:

`defaultAppearance = "light"
autoSwitchAppearance = false
showAppearanceSwitcher = true
`

When I develop locally and view it on localhost, it indeed appears in light mode. but upon deploying, it switches to dark mode on my Mac with OS preferences for dark mode. 

I'm hoping that I'm reading [this page](https://jpanther.github.io/congo/docs/version-2/upgrade/) correctly and using default and autoSwitchAppearance right. 

When I change my Mac OS preferences from light to dark, it switches to light mode.  Thanks for any insights!!!

**Comment by jpanther** [comment]
Thanks for picking up on this one @benmatheson. This is not the intended behaviour so this is a bug. The params you are using are the correct ones for the behaviour you seek but there's a logic error that is preventing it from working. I'll open an issue for this and fix it in the next update.

**Reply by benmatheson** [reply]
Awesome, thanks for taking a look.

**Reply by benmatheson** [reply]
I upgraded to 2.1.3 and I think it's working great, thanks @jpanther!
